### PR TITLE
Display the full path of the defaeult prebuild sandbox

### PR DESCRIPTION
**Pull request by cnadeau**

### CODE REVIEW

File: lib/command/config.rb [hunk 1]
```diff
@@ -34,11 +34,11 @@ def cache_path
     end
 
     def prebuild_sandbox_path
-      @dsl_config[:prebuild_sandbox_path] || @deprecated_config["prebuild_path"] || "_Prebuild"
+      @dsl_config[:prebuild_sandbox_path] || @deprecated_config["prebuild_path"] || "#{Dir.pwd}/_Prebuild"
     end
 
     def prebuild_delta_path
-      @dsl_config[:prebuild_delta_path] || @deprecated_config["prebuild_delta_path"] || "_Prebuild_delta/changes.json"
+      @dsl_config[:prebuild_delta_path] || @deprecated_config["prebuild_delta_path"] || "#{Dir.pwd}/_Prebuild_delta/changes.json"
```
Comment (daxiosreview) [inline lib/command/config.rb:41]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/grab/cocoapods-binary-cache/blob/89bcaa56b63e6a96f7f3da7770eb000570ad4cb2/lib/command/config.rb#L41"></span>Layout/LineLength: Line is too long. [129/120]</td>
    </tr>
  </tbody>
</table>

**Comment by trinhngocthuyen** [comment]
Thanks for the MR. I wonder what lead you to this change. Was there any issue with the current implementation that we need to be aware of?

**Comment by cnadeau** [comment]
@trinhngocthuyen I was running in fastlane and I lost an hour before I realized that the _Prebuild was ignored because the prebuild command was run in the fastlane folder, not in the root of the project where the Podfile is. I would have seen it immediately if that log has been in place at that time. Nothing to report other than helping anyone getting in the same situation in the future

**Comment by trinhngocthuyen** [comment]
@cnadeau Thanks for sharing the context.

I would like to express my concern about this change:

- I understand that `fastlane` runs its commands in its folder which caused the issue. However, this issue is not specific to the plugin as it can happen to any other use cases. As an example, running normal `pod install` (without the plugin) also requires the current working dir to be in the project dir (not the `fastlane` dir). Running CocoaPods installation would also lead to the same issue.
- Given that, I think prepending `Dir.pwd` to the `prebuild_sandbox_path` should not fix the problem. And we need to revise many other places if we do it this way.
- What we can do in this case is to surface the issue for investigation by printing the `prebuild_sandbox_path`. In fact, we did print the prebuild sandbox path in https://github.com/grab/cocoapods-binary-cache/blob/master/lib/cocoapods-binary-cache/hooks/pre_install.rb#L57. But it's only visible with the `--verbose` option.
--> Perhaps we can change `Pod::UI.message` to `Pod::UI.puts`, I may suggest.

Your thoughts?

**Pull request closed without merging**